src/main/java/com/labs/introtoprogramming/lab4/image/formats/bmp/BMPParser.java

```diff
@@ -0,0 +1,56 @@
+package com.labs.introtoprogramming.lab4.image.formats.bmp;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Arrays;
+
+public class BMPParser {
+
+  public static int parseHeader(byte[] headerInfo) throws Exception {
+    if (headerInfo.length != 14) {
+      throw new Exception("File format is not a BMP");
```

Comment:
There are two possible ways to make this better:
- Use `RuntimeException`
- Create and use `BMPParseException` or `ImageLoadException` or ...